var/spack/repos/builtin/packages/py-crossmap/package.py

```diff
@@ -36,4 +36,6 @@ class PyCrossmap(PythonPackage):
 
     depends_on('python@2.7:2.7.999', type=('build', 'run'))
     depends_on('py-cython@0.17:', type='build')
-    depends_on('py-pysam',        type='build')
+    depends_on('py-pysam', type=('build', 'run'))
+    depends_on('py-setuptools', type='build')
+    depends_on('py-bx-python', type='run')
```

Comment:
I would also make this `type=('build', 'run')`. If it isn't present at build-time, setuptools will download and install it for you.